src/styles/index.scss

```diff
@@ -1,3 +1,208 @@
+html {
+  font-family: Roboto, sans-serif, Arial;
```

Comment:
The font-family declaration should list generic family names last. Consider changing the order to 'Arial, sans-serif, Roboto'.